cinnabot.go

```diff
@@ -92,6 +98,10 @@ func InitCinnabot(configJSON []byte, lg *log.Logger) *Cinnabot {
 
 	cb := &Cinnabot{Name: cfg.Name, bot: bot, log: lg, keys: cfg}
 	cb.fmap = cb.getDefaultFuncMap()
+	cb.db = model.InitializeDB()
+	cb.cache = cache.New(1*time.Minute, 2*time.Minute)
+	//tag alternates with tag description
+	cb.allTags = []string{"everything", "EVERY tag!! Only for the daring", "events", "EVENTS of cinnamon college", "food", "Free/not free FOOD updates of all kind for the hungry", "weather", "Weather updates. Im not sure why you would want it actually.", "warm", "If you want some nice warm things occasionally"}
```

Comment:
Ah right. After I tried it again I realised its because I wanted the "everything" tag to be at the top. Iterating through keys in a map is not done in the order that it is declared in.

Reply:
I imagine there's no "everything" option anymore?

Reply:
Right